python_modules/dagster/dagster_tests/asset_defs_tests/test_asset_graph.py

```diff
@@ -385,13 +334,16 @@ def test_partitioned_source_asset():
     def downstream_of_partitioned_source():
         pass
 
-    asset_graph = AssetGraph.from_assets([partitioned_source, downstream_of_partitioned_source])
+    asset_graph = asset_graph_from_assets([partitioned_source, downstream_of_partitioned_source])
+
+    if isinstance(asset_graph, ExternalAssetGraph):
+        pytest.xfail("not supported with ExternalAssetGraph")
```

Comment:
I didn't dig in. `assert asset_graph.is_partitioned(AssetKey("partitioned_source"))` is False with an external graph

Reply:
(previously this test was only on internal)